rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/qos_converter.hpp

```diff
@@ -13,8 +13,8 @@
 // limitations under the License.
 
 
-#ifndef QOS_CONVERTER_HPP_
-#define QOS_CONVERTER_HPP_
+#ifndef RMW_FASTRTPS_SHARED_CPP__QOS_CONVERTER_HPP_
```

Comment:
Moved `qos_converter` from `src/` to `/include` making it a public header file. I had previously made it private but now it needs to be accessed by `topic_cache.hpp` and hence has been moved here.

Reply:
It's not clear from the file names which function is where, so I think that having everything in one file will be clearer.